tests/testthat/test-vivax.R

```diff
@@ -0,0 +1,26 @@
+test_that('Test falciparum switch produces same', {
+  parameters_def <- get_parameters()
+  parameters_fal <- get_parameters(parasite = "falciparum")
+  expect_identical(parameters_def, parameters_fal)
+})
+
+test_that('Test vivax model runs', {
+  vivax_parameters <- get_parameters(parasite = "vivax")
+  sim_res <- run_simulation(timesteps = 100, parameters = vivax_parameters)
+  expect_equal(nrow(sim_res), 100)
+})
```

Comment:
Is this test required? I expect there are other tests that run the model with vivax parameters that would fail if this one did?